src/main/target/CC3D/target.h

```diff
@@ -72,8 +72,13 @@
 #define USE_VCP
 #define USE_UART1
 #define USE_UART3
+
+#ifdef CC3D_OPBL
```

Comment:
One `#ifdef` would concentrate all the changes for OPBL into one place (ideally at the very end of the file), so it would read like:

```
----
[stuff for CC3D]
#ifdef CC3D_OPBL
[changes for CC3D_OPBL]
#endif
---
```

Now we are having 2 sections for OPBL. But imagine if we continue this pattern and get to 10 sections for OPBL at some stage...